MonoGame.Framework.Content.Pipeline/FbxImporter.cs

```diff
@@ -21,5 +22,15 @@ public override NodeContent Import(string filename, ContentImporterContext conte
 
             return importer.Import(filename, context);
         }
+
+        public override NodeContent Import(Stream input, string virtualFilename, ContentImporterContext context)
```

Comment:
It's a public API, so it should have the XML documentation.  Some of the existing public APIs may be missing them, but new public APIs must have them.